service/src/main/java/uk/nhs/adaptors/gp2gp/tasks/GetGpcDocumentTaskDefinition.java

```diff
@@ -0,0 +1,16 @@
+package uk.nhs.adaptors.gp2gp.tasks;
```

Comment:
Does the package "task" describe a feature of the application as per our package-by-feature convention?

The Gpc task definitions and executors should be in a "gpc" package. They related to the GP Connect (GPC) feature of the GP2GP adaptor.

The superclasses and factories TaskDefinition, TaskExecutor belong in `common.task` because these relate to all tasks defined in system.